### PR TITLE
Wvw

### PR DESCRIPTION
**Pull request by werdes**

### CODE REVIEW

File: Gw2Sharp/WebApi/V2/Clients/Wvw/WvwClient.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    public class WvwClient : Gw2WebApiBaseClient, IWvwClient
+    {
+        private readonly IWvwAbilitiesClient abilities;
+        private readonly IWvwMatchesClient matches;
+        private readonly IWvwObjectivesClient objectives;
+        private readonly IWvwRanksClient ranks;
+        private readonly IWvwUpgradesClient upgrades;
+
+        /// <summary>
+        /// Creates a new <see cref="WvwClient"/> that is used for the API v2 commerce endpoint.
```
Comment (Archomeda) [inline Gw2Sharp/WebApi/V2/Clients/Wvw/WvwClient.cs:18]: I'm sure that this is not the commerce endpoint 😄

**Review by Archomeda — CHANGES_REQUESTED** [review]
Thanks for your contribution!

There's something I'd like to address before merging:
Can you recheck all xmldoc comments and make sure that they have correct casing that are in line with the other comments? E.g. starting with a uppercase letter, WvW properly cased, id(s) fully lowercase, ending with a dot, etc.

Additionally, I'll have to think about the naming of WvwMatchWorldIdContainer and WvwMatchWorldValueContainer a bit more. I'm sure I've used similar models that are referenced from multiple places (instead of just one reference), but I can't remember it on the top of my head where exactly.

**Comment by werdes** [comment]
> Additionally, I'll have to think about the naming of WvwMatchWorldIdContainer and WvwMatchWorldValueContainer a bit more. I'm sure I've used similar models that are referenced from multiple places (instead of just one reference), but I can't remember it on the top of my head where exactly.

Yeah i thought about that too. Another way to display those would be a `Dictionary<WvwOwner,  int>`, but i think that would not fit the rest of your inplementation, would it?

And damn, i missed that one commerce comment, that‘s what i get for copy/paste

**Review by Archomeda — APPROVED** [review]
I've gone over the code again, and made some additional changes. I also went ahead and added the subendpoints under /v2/wvw/matches into this PR.

Given that the guild and team stats are still broken on the API, I call this WvW PR feature complete. After merging, I'll make sure to update all documentation related files in another PR.

Thanks again!

**Pull request merged**